scene/gui/gradient_edit.cpp

```diff
@@ -147,7 +147,6 @@ void GradientEdit::_gui_input(const Ref<InputEvent> &p_event) {
 		grabbed = _get_point_from_pos(x);
 		//grab or select
 		if (grabbed != -1) {
-			grabbed = false;
```

Comment:
Looks like a typo, it's likely meant to be `grabbing = false;`, no?

Reply:
I'll check here.

Reply:
If I change to `grabbing = false` I can't grab anymore.